nix/haskell-packages.nix

```diff
@@ -30,7 +31,7 @@ let
   metricOverlay      = import ./overlays/metric.nix       { inherit pkgs; };
 
   activeOverlays = [ requiredOverlay ]
-      ++ optional enablePhaseMetrics metricOverlay
+  ##  ++ optional enablePhaseMetrics metricOverlay  # Removed temporarily
```

Comment:
undo this and switch nixpkgs version to `f5afebd42b740720beed076bfbf233a6f64a047d` on our fork.